### PR TITLE
MEN-7508: Add a troubleshoot entry for the `fsck` error

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Review by estenberg — APPROVED** [review]
Nice and clear!

File: 301.Troubleshoot/03.Mender-Client/docs.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ taxonomy:
...
+When running `mender-artifact write -f ssh://...` command to create a Mender Artifact from a virtual
+device, you can get the following error:
+
+```
+runFsck error: fsck error: exit status 12
+```
+
+Recent releases of the virtual device use new filesystem features, which requires up to date
+filesystem tools (namely `fsck`) on the host. Debian 12, Ubuntu 24.04 and newer OSs work correctly.
+
```
Comment (TheMeaningfulEngineer) [inline 301.Troubleshoot/03.Mender-Client/docs.md:18]: I cant make a suggestion with code blocks.
Here is it in raw form:

---

You can workaround the issue by executing a few extra steps:

On the shell **of the device** execute:

```
mkfifo /data/mender/dump
mender-snapshot dump > /data/mender/dump
```

Open another shell, **on the host**, where you have the variables form the getting started and run:

```
ssh -p 8822 root@${IP_ADDRESS} cat /data/mender/dump > rootfs.ext4
mender-artifact write rootfs-image \
    -f ssh://"${USER}@${IP_ADDRESS}" \
    -t "${DEVICE_TYPE}" \
    -n system-v1 \
    -o system-v1.mender \
    -f rootfs.ext4
```

You can upload the `system-v1.mender` and continue with the evaluation.

**Review by TheMeaningfulEngineer — CHANGES_REQUESTED** [review]
Made the suggestions for a workaround

**Review by TheMeaningfulEngineer — APPROVED** [review]

**Comment by TheMeaningfulEngineer** [comment]
Forgot to add.
I rendered this and executed the steps from the rendered docs.
The system-v1.mender was created correctly and I did an update with it.

**Comment by lluiscampos** [comment]
> Forgot to add. I rendered this and executed the steps from the rendered docs. The system-v1.mender was created correctly and I did an update with it.

Thank you very much! I was waiting for the dev branch to be updated before asking for merges. But once a human has checked it, I can do:

@mender-test-bot cherry-pick to:
* hosted
* 3.7.x

**Pull request merged**